frontend/src/modules/projects/views/projectView.js

```diff
@@ -75,28 +75,16 @@ define(function(require) {
 
     deleteProjectPrompt: function(event) {
       event && event.preventDefault();
-      if(this.model.get('_isShared') === true) {
-        if(this.model.get('createdBy') === Origin.sessionModel.id){
-          Origin.Notify.confirm({
-            type: 'warning',
-            title: Origin.l10n.t('app.deletesharedproject'),
-            text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.confirmdeletesharedprojectwarning'),
-            destructive: true,
-            callback: _.bind(this.deleteProjectConfirm, this)
-          });
-        } else {
-          Origin.Notify.alert({
-            type: 'error',
-            text: Origin.l10n.t('app.errorpermission')
-          });
-        }
-        return;
+      var isAdmin = Origin.permissions.hasPermissions(['*/*:create','*/*:read','*/*:update','*/*:delete']);
+      var isShared = this.model.get('_isShared') === true;
+      var titleKey = isShared ? 'deletesharedproject' : 'deleteproject';
+      var textKey = isShared ? 'confirmdeletesharedprojectwarning' : 'confirmdeleteprojectwarning';
 
-      }
       Origin.Notify.confirm({
         type: 'warning',
-        title: Origin.l10n.t('app.deleteproject'),
-        text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.confirmdeleteprojectwarning'),
+        title: Origin.l10n.t('app.' + titleKey),
+        text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.' + textKey),
+        destructive: isShared && !isAdmin, // waive the wait for 'trusted' admin users
```

Comment:
Hi @lc-thomasberger we discussed this in the stand-up yesterday and we think that we should keep this wait as the visibility of some courses is changing with this release. It's safer to keep this for now.

Reply:
Hello, the whole PR?
If we do not merge this PR we must also remove the option "share all courses" for the user delete dropdown.

Reply:
No not the whole PR. It's fine for an admin to be able to delete any course, but we want the admin to have the 5 second wait before they can delete it